### ISSUE TITLE
Payment systems that can be used in Turkey should be added

### ISSUE THREAD

**Issue by LegoCraftNW** [CraftingStore/Suggestions#678]
Users in Turkey cannot use your system because there is no valid payment system in Turkey. Payment systems in Turkey can also receive foreign currency payments. If a few payment systems valid in Turkey are added, your Turkish users will increase. Currently, most Turkish game server owners receive their payments using paid web scripts and these scripts are not cheap at all.

I also want to use your system for my Rust server, but there is no payment system on your site where I can receive payment in TL. 

Several Turkish payment systems:
Shopier - https://www.shopier.com/
Shipy - https://shipy.net/
iyzico - https://www.iyzico.com/
Paytr - https://www.paytr.com/
Paywant - https://www.paywant.com/

**Comment by Tim-NL** [comment]
Hi,

We support various payment methods that support the Turkish lira;

https://stripe.com/docs/currencies
https://docs.paymentwall.com/reference/currencies

Both of those payment gateways offer 20+ payment methods each, including local ones.

And we offer the ability to create a custom gateway on our Platinum plan, so you can program your own: https://help.craftingstore.net/payment-gateways/custom-payment-method-gateway

As you can already do payments using the Lira, we do not see the need to implement the provided payment methods.

-Tim

**Comment by LegoCraftNW** [comment]
> Hi,
> 
> We support various payment methods that support the Turkish lira;
> 
> https://stripe.com/docs/currencies https://docs.paymentwall.com/reference/currencies
> 
> Both of those payment gateways offer 20+ payment methods each, including local ones.
> 
> And we offer the ability to create a custom gateway on our Platinum plan, so you can program your own: https://help.craftingstore.net/payment-gateways/custom-payment-method-gateway
> 
> As you can already do payments using the Lira, we do not see the need to implement the provided payment methods.
> 
> -Tim

Yes, these systems you mentioned can allow me to receive payment in TL. But these systems require you to have a company in order to receive payment. I want to receive payment before starting a company. Even if I set up a budget to establish a company, I will not have any income to continue. I'm just going to start this business, I have only 1 server and domain name. That's why I wanted a simpler system. Paypal is not in Turkey, so there is no suitable platform where I can receive the payment.

**Comment by Tim-NL** [comment]
Hi,

You don't need to have a business for either Stripe or Terminal3 (Paymentwall). You can use them as a person without any issues.

-Tim

**Comment by LegoCraftNW** [comment]
> Hi,
> 
> You don't need to have a business for either Stripe or Terminal3 (Paymentwall). You can use them as a person without any issues.
> 
> -Tim

Paymentwall wants a lot of information for the individual, I think I filled them out and I'm waiting for the result, but I heard from you for the first time that stripe can be used individually, if possible, it will be very good for me, thank you for informing me. I hope I can install it without problems. I don't know if TRY currency was displayed on your site, but it would be great if it did.

**Comment by LegoCraftNW** [comment]
Stripe can receive payment in TRY, but when I create an account, I cannot find Turkey in the country list, so it does not work actively in Turkey. When I choose another country, it asks for address information and I do not have an address in those countries. So stripe is not possible for me. I will try paymentwall payment system which is my last chance

https://stripe.com/global
